.gitignore

```diff
@@ -40,6 +40,10 @@ vendor/
 # TypeScript files
 tsconfig.tsbuildinfo
 
+# WooCommerce Plugin built files
```

Comment:
Is this necessary, given that it's ignored in `plugins/woocommerce/.gitignore` already? At first I was thinking maybe we should centralize it all in the root `.gitignore`, but I think for project-specific ignores, it makes the most sense to live in the folder, so that it's present in the mirror repository.